### ISSUE TITLE
[FEATURE REQUEST] improve control panel of CAD with accessibility in mind

### ISSUE THREAD

**Issue by gregjozk** [mrdokenny/Cookie-AutoDelete#1125]
### Acknowledgements

- [X] I acknowledge that I have read the above items

### Is your feature request related to a problem? Please describe.

Control panel of a settings panel and other UI elements of CAD are not accessible for screen readers, so blind people cannot indipendently interact with them. Tested mostly inFirefox with NVDA.

### Describe the solution that you'd like

Redesign them with changing clickable elements with regular checkboxes and other screen reader friendly elements and properly tagging them.

### Additional Context

Hello,

because I'm blind, I'm using screen readers when I'm working with computers. It means, that due to my condition, I cannot see changes on the screen, when I activate controls on a control panel of CAD. As Isee, there are most controls, which could be changed with regular checkboxes instead of clickable elements. When checkbox is  checked,  screen reader announce it, but when element is clickable, screen readercannot report a state of this element and I cannot know if this element is active or not. Ideal solution would be if designers of UI would follow latest WCAG (Web content accessibility guidelines), recommended is level AA.
If you want test how screen readers read web pages, here are listed some of them:
- MAC/iOS: VoiceOver
- Android:Talkback
- Chrome OS: Chromevox
- Linux: Orca (sourcecode available in a gnome project on gitlub)
- Windows: Narrator, NVDA (Sourcecode available on github), JAWS and Supernova (last 2 are very expensive)
So I suggest, if it is possible, can you redesign your GUI to be compatible with screen readers, so blind people would also interact with switches and other elements of settings and other panels.

Thank you.

Regards,
Jožef

**Comment by kennethtran93** [comment]
Thank you for the request.  I will look into it as i continue to enhance the project.
